src/app/pages/IdxPage/index.jsx

```diff
@@ -75,7 +77,7 @@ const IdxPage = ({
         openGraphType="website"
       />
       <LinkedData type="WebPage" seoTitle={seoTitle} />
-      <main role="main">
+      <main role="main" {...mainLangAttribute}>
```

Comment:
Do we need to pass the lang attribute to the `main` element? We set the `html` lang to `ru-UA` already, and then set a different lang on the header and footer. Are we ok for this to just inherit the `lang` attribute of the page? Let me know if I've missed something!

Reply:
@tochwill I was also wondering the same, but I wasn't sure if the main would inherit it automatically.
However, it would make sense that everything on the page uses the lang set in the `HTML`, rather than having to set it again if the element is in between elements which use other lang.

We can remove it for now and double-check during the a11y swarm.